framework-test/src/main/java/org/checkerframework/framework/test/diagnostics/TestDiagnosticUtils.java

```diff
@@ -22,13 +22,23 @@ public class TestDiagnosticUtils {
 
     /** How the diagnostics appear in Java source files. */
     public static final String DIAGNOSTIC_IN_JAVA_REGEX =
-            "\\s*(error|fixable-error|warning|fixable-warning|other):\\s*(\\(?.*\\)?)\\s*";
+            "\\s*(error|fixable-error|warning|fixable-warning|other):\\s*(\\(?[\\s\\S]*\\)?)\\s*";
```

Comment:
What this now allows is that in an expected error like `// :: error: (what.I.expect)` there can now definitely be a newline character (`.` does not necessarily include newlines).
Do you have an example test where this happens?
I would rather expect that we can limit this to `[\\S]`, to only allow non-whitespace characters in the expected error message.
Either way, a comment about the motivation would be good here, not on the pattern below.